protsushyn-exam.v.2.0.css

```diff
@@ -166,12 +189,26 @@ section:nth-of-type(1) h1 a {
 
 .three-image {
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
+    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
     justify-items: center;
     margin: 0 auto;
     grid-gap: 25px;
 }
 
+@media screen and (max-width: 400px) {
```

Comment:
@Mmarvinn In your breakpoints try to follow the `bootstrap` recommendation https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints.

Reply:
yes of curse, i will take it